docs/tasks/step-3.md

```diff
@@ -2,20 +2,57 @@
 
 ## a) Add table component
 
-On the `/all-books` page list all of the books using a [vuetify data table](https://vuetifyjs.com/en/components/data-tables/basics/).
+In this section, we will be adding a list of all books to the `all-books.vue` layout, using a [vuetify data table](https://vuetifyjs.com/en/components/data-tables/basics/). Similar to the book card, component, we will add the component using kebab-case naming standard, by adding the following to the template:
```

Comment:
` all-books.vue` page?

Reply:
as in change the wording 'ayout' to 'page' ?